File: scratch/list_tables.py
import re

file_path = r'c:\xampp\htdocs\laravel\dcdhaka.srsibd.com-2-main\upms\almsgov_upms.sql'
tables = []
with open(file_path, 'r', encoding='utf-8', errors='ignore') as f:
    for line in f:
        match = re.search(r'CREATE TABLE `([^`]+)`', line)
        if match:
            tables.append(match.group(1))

print("Found tables:", tables)

File: scratch/list_tables.php
<?php

$filePath = 'c:\xampp\htdocs\laravel\dcdhaka.srsibd.com-2-main\upms\almsgov_upms.sql';
$handle = fopen($filePath, 'r');
$tables = [];

if ($handle) {
    while (($line = fgets($handle)) !== false) {
        if (preg_match('/CREATE TABLE `([^`]+)`/', $line, $matches)) {
            $tables[] = $matches[1];
        }
    }
    fclose($handle);
}

echo "Found tables: " . implode(', ', $tables) . "\n";
